exercism/hamming/hamming.js

```diff
@@ -0,0 +1,18 @@
+var Hamming = function() {}
+
+Hamming.prototype.compute = function (string1, string2) {
+  var counter = 0
+
+  if(string1.length !== string2.length) {
+    throw "DNA strands must be of equal length."
```

Comment:
the test will be expecting you to throw a new error, not just throw an exception.
throw new Error("DNA strands must be of equal length.")